source/manage-apps/configure/config/sync.txt

```diff
@@ -215,88 +219,160 @@ Sync configuration:
        - ``"enabled"``
        - ``"disabled"``
 
-   * - | ``permissions.rules.<Coll>``
-       | String
-     - You can define rules that apply to a single collection by specifying
-       the collection name. App Services apps apply rules in order from the beginning
-       of the array, so start with the most specific rules first.
+   * - | ``permissions.rules.<Type Name>``
+       | Array<Role>
+     
+     - You can define a set of roles that apply to a specific object
+       type. These roles apply in order from the beginning of the array,
+       so define the most specific rules first.
 
-   * - | ``rules.<Coll>.name``
+   * - | ``rules.<Type Name>.[].name``
        | String
-     - The name of the specific rule in the rules array. You might use a 
-       descriptive name to make it easier to parse the rules, such as 
-       "adminReadWrite" or "employeeReadOnly".
 
-   * - | ``rules.<Coll>.applyWhen``
-       | JSON Expression
-     - An :ref:`expression <expressions>` that evaluates to ``true`` when 
-       the rule should be applied. An example might be 
-       ``{"%%user.custom_data.isGlobalAdmin": true}``. This reads the 
-       value of the ``isGlobalAdmin`` bool from the :ref:`user's custom data 
-       <custom-user-data>`, and applies the rule when the value of this 
-       bool is ``true``.
+     - The name of the role. You might use a descriptive name, such as
+       ``"adminReadWrite"`` or ``"employeeReadOnly"``, to make it easier
+       to understand the rules.
 
-   * - | ``rules.<Coll>.read``
-       | JSON Expression
-     - An :ref:`expression <expressions>` that evaluates to ``true`` when a 
-       user has permission to read objects in the collection. If the 
-       expression evaluates to ``false``, {+service-short+} does not allow 
-       the user to read an object or its properties.
-
-   * - | ``rules.<Coll>.write``
-       | JSON Expression
-     - An :ref:`expression <expressions>` that evaluates to ``true`` when a 
-       user has permission to write objects in the collection. If the 
-       expression evaluates to ``false``, {+service-short+} does not allow 
-       the user to modify an object or its properties. Write permission
-       requires read permission. A user cannot write to a document that
-       they cannot read.
+   * - | ``rules.<Type Name>.[].applyWhen``
+       | Expression
+     
+     - An :ref:`expression <expressions>` that evaluates to ``true``
+       when the role applies to the user. 
+       
+       .. example::
```

Comment:
I broke this table up a bit with some headings. This makes the first column less wide + I tweaked the widths slightly too so that the description gets more space.

Reply:
Nice! I think this is an improvement - thanks for tweaking it!